flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/HeapInternalTimerService.java

```diff
@@ -49,13 +50,13 @@
 	/**
 	 * Processing time timers that are currently in-flight.
 	 */
-	private final Set<InternalTimer<K, N>>[] processingTimeTimersByKeyGroup;
+	private final Map<String, InternalTimer<K, N>>[] processingTimeTimersByKeyGroup;
```

Comment:
@StefanRRichter 
I switch from set to map because in deleteProcessingTimeTimer/deleteEventTimeTimer method, I have to check if contais Timer with currentKey, namespace and time while I don't have version info.
If I use Set as interface, then I have to iterator Set to fetch exactly timer object, which is not convenient as Map